src/v3/src/transformer/password/transformEnrollPasswordAuthenticator.ts

```diff
@@ -213,22 +213,5 @@ export const transformEnrollPasswordAuthenticator: IdxStepTransformer = ({
     },
   };
 
-  dataSchema.confirmPassword = {
-    validate: (data: FormBag['data']) => {
-      const newPw = data[passwordFieldName] as string;
-      const confirmPw = data.confirmPassword;
-      const errorMessages: WidgetMessage[] = [];
-      if (newPw && newPw !== confirmPw) {
-        errorMessages.push({
-          name: 'confirmPassword',
-          class: 'ERROR',
-          message: loc('password.enroll.error.match', 'login'),
-          i18n: { key: 'password.enroll.error.match' },
-        });
-      }
-      return errorMessages.length > 0 ? errorMessages : undefined;
-    },
-  };
```

Comment:
This logic is already defined [here](https://github.com/okta/okta-signin-widget/pull/3162/files#diff-e24d8f2381697a352414e8af1c48a5aa7bf0b8f41680c49edf37a510b747773cL204-L211).